mmv1/templates/terraform/examples/document_ai_warehouse_document_schema_text.tf.erb

```diff
@@ -0,0 +1,203 @@
+resource "google_document_ai_warehouse_document_schema" "example_text" {
+  project_number     = "406397197218"
+  display_name       = "test-property-text"
+  location           = "us"
+  document_is_folder = false
+
+  property_definitions {
+    name                 = "prop3"
+    display_name         = "propdisp3"
+    is_repeatable        = false
+    is_filterable        = true
+    is_searchable        = true
+    is_metadata          = false
+    is_required          = false
+    retrieval_importance = "HIGHEST"
+    schema_sources {
+      name           = "dummy_source"
+      processor_type = "dummy_processor"
+    }
+    text_type_options {}
+  }
+}
+
+resource "google_document_ai_warehouse_document_schema" "example_integer" {
+  project_number = "406397197218"
+  display_name   = "test-property-integer"
+  location       = "us"
+
+  property_definitions {
+    name                 = "prop1"
+    display_name         = "propdisp1"
+    is_repeatable        = false
+    is_filterable        = true
+    is_searchable        = true
+    is_metadata          = false
+    is_required          = false
+    retrieval_importance = "HIGHEST"
+    schema_sources {
+      name           = "dummy_source"
+      processor_type = "dummy_processor"
+    }
+    integer_type_options {}
+  }
+}
+
+resource "google_document_ai_warehouse_document_schema" "example_float" {
+  project_number = "406397197218"
+  display_name   = "test-property-float"
+  location       = "us"
+
+  property_definitions {
+    name                 = "prop2"
+    display_name         = "propdisp2"
+    is_repeatable        = false
+    is_filterable        = true
+    is_searchable        = true
+    is_metadata          = false
+    is_required          = false
+    retrieval_importance = "HIGHEST"
+    schema_sources {
+      name           = "dummy_source"
+      processor_type = "dummy_processor"
+    }
+    float_type_options {}
+  }
+}
+
+resource "google_document_ai_warehouse_document_schema" "example_property" {
+  project_number     = "406397197218"
```

Comment:
project_number should not be hardcoded here. It can be `data.google_project.project.number`